api/apps/services/file_api_service.py

```
@@ -555,8 +566,8 @@ def _move_entry_recursive(source_file_entry, dest_folder_entry, override_name=No
                     dest_folder_entry.id,
                     new_location,
                 )
-            except Exception as storage_err:
-                raise RuntimeError(f"Move file failed at storage layer: {str(storage_err)}")
+            except Exception as storage_err:  # noqa: BLE001 - re-raised as RuntimeError with context; storage backends raise SDK-specific errors
+                raise RuntimeError(f"Move file failed at storage layer: {storage_err!s}")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)